[2022-01-27 14:50:47,799] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: cgi_40.load_data_source 2022-01-27T14:50:46.505139+00:00 [queued]>
[2022-01-27 14:50:47,808] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: cgi_40.load_data_source 2022-01-27T14:50:46.505139+00:00 [queued]>
[2022-01-27 14:50:47,808] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-01-27 14:50:47,808] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-01-27 14:50:47,808] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-01-27 14:50:47,817] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): load_data_source> on 2022-01-27T14:50:46.505139+00:00
[2022-01-27 14:50:47,825] {standard_task_runner.py:52} INFO - Started process 114 to run task
[2022-01-27 14:50:47,833] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cgi_40', 'load_data_source', '2022-01-27T14:50:46.505139+00:00', '--job-id', '18', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/cgi40_dag.py', '--cfg-path', '/tmp/tmp7d8dsrwr', '--error-file', '/tmp/tmp47g97dn3']
[2022-01-27 14:50:47,833] {standard_task_runner.py:77} INFO - Job 18: Subtask load_data_source
[2022-01-27 14:50:47,859] {logging_mixin.py:104} INFO - Running <TaskInstance: cgi_40.load_data_source 2022-01-27T14:50:46.505139+00:00 [running]> on host bb053c5f0074
[2022-01-27 14:50:47,894] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=oulespaul@gmail.com
AIRFLOW_CTX_DAG_OWNER=ETDA
AIRFLOW_CTX_DAG_ID=cgi_40
AIRFLOW_CTX_TASK_ID=load_data_source
AIRFLOW_CTX_EXECUTION_DATE=2022-01-27T14:50:46.505139+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-27T14:50:46.505139+00:00
[2022-01-27 14:50:47,895] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-01-27 14:50:47,895] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'cd /opt && ls']
[2022-01-27 14:50:47,903] {subprocess.py:75} INFO - Output:
[2022-01-27 14:50:47,906] {subprocess.py:79} INFO - ***
[2022-01-27 14:50:47,906] {subprocess.py:79} INFO - microsoft
[2022-01-27 14:50:47,906] {subprocess.py:79} INFO - mssql-tools
[2022-01-27 14:50:47,907] {subprocess.py:83} INFO - Command exited with return code 0
[2022-01-27 14:50:47,925] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=cgi_40, task_id=load_data_source, execution_date=20220127T145046, start_date=20220127T145047, end_date=20220127T145047
[2022-01-27 14:50:47,941] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-27 14:50:47,960] {local_task_job.py:151} INFO - Task exited with return code 0
